Scaffolding postgre database failed: Could not compile data connection assembly

Hello,

It seems that NP cannot compile my database: Postgre 14
If I open my postgre database in LinkPad it works (windows), if I do the same in NetPad (macos) it can't. I also tested my connection and it succeeded. Not sure what I'm missing, maybe this part is failing : https://github.com/tareqimbasher/NetPad/blob/5be4889efc4abe037934eabf7791a3ae0fb61889/src/Infrastructure/NetPad.Infrastructure/Data/EntityFrameworkCore/EntityFrameworkResourcesGenerator.cs#L231

![image](https://github.com/tareqimbasher/NetPad/assets/23088305/08127adb-a7cd-469e-962b-2965a3a911d3)

```bash
[500] PATCH: http://localhost:57930/data-connections/34e80e5e-b87c-4416-b060-ddfa9e9db4e3/database-structure
```

```js
{
  details: null,
  message: "Could not compile data connection assembly. Compilation failed with the following diagnostics: \n(39,26): error CS0246: The type or namespace name 'Equipment_HIDDEN' could not be found (are you missing a using directive or an assembly reference?)"
}
```

--- a/src/Infrastructure/NetPad.Infrastructure/Data/EntityFrameworkCore/EntityFrameworkResourcesGenerator.cs
+++ b/src/Infrastructure/NetPad.Infrastructure/Data/EntityFrameworkCore/EntityFrameworkResourcesGenerator.cs
@@ -237,7 +237,7 @@ public class EntityFrameworkResourcesGenerator : IDataConnectionResourcesGenerat
     public static Microsoft.EntityFrameworkCore.DbSet<{entityType}> {propertyName} => DataContext.{dbContextPropertyName};");
 
             // Rename property on DbContext
-            dbContextCodeLines[iLine] = line.Replace(propertyName, dbContextPropertyName);
+            dbContextCodeLines[iLine] = line.Replace($" {propertyName} ", $" {dbContextPropertyName} ");
         }
 
         // Replace the DbContext code since we modified it above when renaming properties

